jsapp/js/components/processing/sidebar/sidebarDisplaySettings.module.scss

```diff
@@ -0,0 +1,46 @@
+@use '~kobo-common/src/styles/colors';
+@use 'scss/sizes';
+
+.root {
+  display: flex;
+  justify-content: right;
+  padding-top: sizes.$x18;
+  padding-bottom: sizes.$x10;
+}
+
+.description {
+  padding-bottom: sizes.$x30;
+}
+
+.display {
+  padding-bottom: sizes.$x40;
+}
+
+.hasTranslation {
+  span::before {
+    content: '(';
+  }
+
+  span::after {
+    content: ')';
+  }
+}
+
+.options {
```

Comment:
Unfortunately it needs `margin: 0 !important` to override the styles from `kobo-modal` - it has some styles for `> ul` and `> p` to work with text content (turns out it was a bad idea :P)